assets/styles/layouts/_default/baseof.css

```diff
@@ -179,12 +185,32 @@
 .page-footer-section__social-link {
     opacity: 0.2;
     color: var(--color-secondary-darker);
+    transition: opacity 0.3s ease;
+}
+
+.page-footer-section__social-link:hover {
+    opacity: 0.4;
 }
 
 .page-footer-section__end {
     opacity: 0.7;
 }
 
+.page-footer-section__end:hover {
+    opacity: 1;
+}
+
 .page-footer-section__made-by-link {
     color: #0000004d;
 }
+
+.sr-only {
```

Comment:
Let's put such global styles somewhere in the root folder of the styles folder (e.g., `assets/styles/screen-reader.css` or into the `theme.css`). It's not limited to the `baseof` template, is it? And maybe add some comment. I somehow remembered `.sr-only`, but still had to search for it.

Reply:
moved